libc/test/src/__support/integer_literals_test.cpp

```diff
@@ -133,3 +133,18 @@ TEST(LlvmLibcIntegerLiteralTest, u256) {
       U256_MAX,
       0xFFFFFFFF'FFFFFFFF'FFFFFFFF'FFFFFFFF'FFFFFFFF'FFFFFFFF'FFFFFFFF'FFFFFFFF_u256);
 }
+
+TEST(LlvmLibcIntegerLiteralTest, parse_bigint) {
+  using T = LIBC_NAMESPACE::Int<128>;
+  struct {
+    const char *str;
+    int expected;
+  } constexpr TEST_CASES[] = {
+      {"0", 0}, {"-1", -1}, {"+1", 1}, {"-0xFF", -255}, {"-0b11", -3},
+  };
+  for (auto tc : TEST_CASES) {
+    T actual = LIBC_NAMESPACE::parse_bigint<T>(tc.str);
+    T expected(tc.expected);
```

Comment:
Could probably elide this statement if you just declared `expected` as `T` rather than `int`.

Reply:
Done in [1afd5a4](https://github.com/llvm/llvm-project/pull/86137/commits/1afd5a42f37a7093750121227f951a7d6fde4de1)